configs/docker-centos-img/local-centos-7-x86_64.cfg

```diff
@@ -65,5 +65,12 @@ enabled=1
 gpgcheck=0
 cost=2000
 
+[stx-cengn-extras]
+name=centos 7 x86_64 - stx-cengn-extras
+#baseurl=<URL of your personal repo> or baseurl=file:///mnt/Localrepo
+baseurl=http://mirror.starlingx.cengn.ca/mirror/starlingx/master/centos/latest_build/inputs/RPMS/
+enabled=1
```

Comment:
So how is this different than the enablement of the stx-cengn repo included in the other commit?

Reply:
One is input to the stx build system and the other is outputs: 
```
[stx-cengn]
name=centos 7 x86_64 - stx-cengn
#baseurl=<URL of your personal repo> or baseurl=file:///mnt/Localrepo
baseurl=http://mirror.starlingx.cengn.ca/mirror/starlingx/master/centos/latest_build/outputs/RPMS/std/
enabled=1
gpgcheck=0
cost=2000

[stx-cengn-extras]
name=centos 7 x86_64 - stx-cengn-extras
#baseurl=<URL of your personal repo> or baseurl=file:///mnt/Localrepo
baseurl=http://mirror.starlingx.cengn.ca/mirror/starlingx/master/centos/latest_build/inputs/RPMS/
enabled=1
gpgcheck=0
cost=2000
```